components/TestLibraryStepper/TestLibraryStepper.tsx

```diff
@@ -0,0 +1,147 @@
+            color: '#fff',
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            display: 'flex',
+            borderRadius: '50%',
+            justifyContent: 'center',
+        },
+        active: {
+            color: 'rgb(255, 193, 0)',
+            fontSize: 18,
+        },
+        current: {
+            color: 'rgb(255, 255, 255)',
+            zIndex: 0,
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            borderRadius: '50%',
+            backgroundColor: 'rgb(255, 193, 0)',
+        },
+        completed: {
+            color: 'rgb(255, 255, 255)',
+            zIndex: 0,
+            fontSize: 18,
+            width: 30,
+            height: 30,
+            borderRadius: '50%',
+            backgroundColor: 'rgb(255, 193, 0)',
+        },
+    });
+
+    function TestSwitchStepIcon(props: StepIconProps) {
+        const classes = TestSwitchIconStyles();
+        const {active, completed} = props;
+        return (
+            <div
+                className={clsx(classes.root, {
+                    [classes.active]: active,
+                })}
+            >
+                {completed ? <Check className={classes.completed}/> : <MoreHoriz className={classes.current}/>}
+            </div>
+        );
+    }
+
+    //overrides for material ui stepper connector styles
+    const TestSwitchConnector = withStyles({
+        active: {
+            '& $line': {
+                borderColor: 'rgb(255, 193, 0)',
+            },
+        },
+        completed: {
+            '& $line': {
+                borderColor: 'rgb(255, 193, 0)',
+            },
+        },
+        line: {
+            borderColor: '#eaeaf0',
+            borderTopWidth: 4,
+            borderRadius: 1,
+            zIndex: -2,
+            lineHeight: 50,
+        },
+    })(StepConnector);
+
+    return (
+        <div className="stepperContainer">
+            <MuiThemeProvider theme={testSwitchTheme}>
+                <Stepper alternativeLabel activeStep={activeStep} connector={<TestSwitchConnector/>}>
+                    {steps.map((label) => (
+                        <Step key={label}>
+                            <StepLabel StepIconComponent={TestSwitchStepIcon} className="stepLabel">{label}</StepLabel>
+                        </Step>
+                    ))}
+                </Stepper>
+            </MuiThemeProvider>
+            <div className="stepperBtnContainer">
+                {activeStep === steps.length ? (
+                    <div>
+                        <Typography align={"center"} className="finished">
+                            All tests completed.
+                        </Typography>
+                    </div>
+                ) : (
+                    <div className="stepperBtnContainer">
+                        <div>
+                            <Typography align={"center"}>
+                                <a className="stepperBtn" href="#"
```

Comment:
We should probably wrap this link in a [Next `Link`](https://nextjs.org/docs/api-reference/next/link).

That will automatically do things like 'pre-fetch' the next page, so make switching between pages seemless.

Reply:
done. ✔️